registry/shows.js

```diff
@@ -115,7 +115,7 @@ shows.package = function (doc, req) {
         // .../_rewrite/pkg/-/pkg-version.tgz
         // or: /pkg/-/pkg-version.tgz
         // depending on what requested path is.
-        var tf = [doc.name, '-', t.split('/').pop()]
+        var tf = [doc.name.replace('/','%2f'), '-', f.replace('/','%2f')]
```

Comment:
This looks fine, but can we just use `encodeURIComponent` instead of regex?

Reply:
Sadly, this breaks publishing scoped modules, as "@" is converted to "%40". I think we only want to escape the "/".